### DISCUSSION TITLE
Is the training.main() doing a training from scratch, or fine-tuned on pre-trained weights?

### DISCUSSION THREAD

**Discussion by wsong1106** [mlfoundations/open_clip#682] [Q&A]
Hi all - 

I am training open_clip on my custom datasets, following the [instructions here](but the training actually was conducted using GPU). After training for 30 epochs, the image_to_text_R@n and the text_to_image_R@n are pretty low. My question is: is the training.main() doing the training from scratch, or is it doing the training from a pre-trained weights?

How can I make training.main() to start training from a pre-trained weights, for example your huggingface checkpoints?

**Comment by mitchellnw** [comment]
It's training from scratch

**Reply by wsong1106** [reply]
Thanks for your response, is there a way that we can start training from CLIP's pre-trained weights?

**Reply by wsong1106** [reply]
I think I figured out, by adding `--pretrained openai` into the commands
